packages/components/src/Sales/Direct/Form.tsx

```diff
@@ -263,6 +271,12 @@ const SalesDirectForm: VFC<Props> = ({
                 placeholder={t('sales.direct.form.quantity.placeholder')}
                 {...register('quantity', {
                   required: t('sales.direct.form.validation.required'),
+                  validate: (value) =>
+                    (parseFloat(value) >= 1 &&
+                      parseFloat(value) <= quantityAvailable?.toNumber()) ||
+                    t('sales.direct.form.validation.in-range', {
+                      quantityAvailable: quantityAvailable?.toNumber(),
+                    }),
```

Comment:
Please avoid to "one-line" this function, very hard to read

Reply:
this was just fixed I by the latest push :)